packages/amplify_auth_cognito/example/integration_test/delete_user_test.dart

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
Do you want to import this into main_test.dart so that it runs with the rest of the test suite?